### ISSUE TITLE
Official weed-fs golang client

### ISSUE THREAD

**Issue by ernado** [chrislusf/weed-fs#6]
I'm using weed-fs from golang project alot with github.com/ginuerzh/weedo and my adapter https://github.com/ernado/weed.
It would be awesome to have official weed-fs client that is fully tested

**Comment by techjanitor** [comment]
I agree completely. There should really be a reference implementation of the API.

**Comment by chrislusf** [comment]
The "official" usage is in the "operation" package, although it has been changing since the APIs are moving.
